### PR TITLE
Add prettier and linting configuration

### PR DESCRIPTION
**Pull request by Vierlip**
- add configuration for prettier and lint
- exclude helm templates (prettier does not support linting Helm templates correctly)
- lint files

### CODE REVIEW

**Review by dkarnutsch — CHANGES_REQUESTED** [review]
The check is missing as GitHub Action. It is only ensured locally.

**Review by kaufmo — COMMENTED** [review]
pipeline for lint is missing?

**Comment by Vierlip** [comment]
> The check is missing as GitHub Action. It is only ensured locally.

72dd07dc45f773515f8f5809061a9efb06d118bd

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Lint and Prettier Check
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
```
Comment (dkarnutsch) [inline .github/workflows/lint.yml:7]: It should also check PRs before merging them

**Review by dkarnutsch — CHANGES_REQUESTED** [review]

**Review by dkarnutsch — APPROVED** [review]

**Pull request merged**